src/meta/src/controller/streaming_job.rs

```diff
@@ -456,7 +452,7 @@ impl CatalogController {
         ensure_job_not_canceled(job_id, &txn).await?;
 
         // Add fragments.
-        let (fragments, actors): (Vec<_>, Vec<_>) = fragment_actors.into_iter().unzip();
+        let (fragments, _): (Vec<_>, Vec<_>) = fragment_actors.into_iter().unzip();
```

Comment:
Since the original `actors` is not used, we may also update `extract_fragment_and_actors_from_fragments` to only return `fragments`, and then we may not need to pass `actor_status` in current method.